pavelib/assets.py

```diff
@@ -20,20 +20,34 @@
 
 ALL_SYSTEMS = ['lms', 'studio']
 COFFEE_DIRS = ['lms', 'cms', 'common']
+
 # A list of directories.  Each will be paired with a sibling /css directory.
 COMMON_SASS_DIRECTORIES = [
-    path("common/static/sass"),
+    (path("common/static/sass"), path("common/static/css"))
+]
+COMMON_SASS_LOOKUP_DIRECTORIES = [
```

Comment:
I think we need more commentary in the code describing the function of the `lookup` directories.